File: app/src/main/java/com/geridea/trentastico/gui/views/requestloader/ParsingErrorMessage.java
package com.geridea.trentastico.gui.views.requestloader;

/*
 * Created with ♥ by Slava on 13/03/2017.
 */
public class ParsingErrorMessage extends AbstractTextMessage {

    public ParsingErrorMessage(int messageId) {
        super(messageId);
    }

    @Override
    public String getText() {
        return "Non sono riuscito ad interpretare gli orari. Riprovo a scaricarli...";
    }
}

File: app/src/main/java/com/geridea/trentastico/gui/views/requestloader/ParsingErrorMessage.kt
package com.geridea.trentastico.gui.views.requestloader

/*
 * Created with ♥ by Slava on 13/03/2017.
 */
class ParsingErrorMessage(messageId: Int) : AbstractTextMessage(messageId) {

    override val text: String
        get() = "Non sono riuscito ad interpretare gli orari. Riprovo a scaricarli..."
}
